src/daft-sql/src/lib.rs

```diff
@@ -316,4 +317,17 @@ mod tests {
 
         Ok(())
     }
+
+    #[rstest]
+    fn test_global_agg(mut planner: SQLPlanner, tbl_1: LogicalPlanRef) -> SQLPlannerResult<()> {
```

Comment:
I'm not totally sure this gets run in `make test` (and running via rust-analyzer didn't work out-of-the-box)